generate.sh

```diff
@@ -6,9 +6,12 @@ if [ ! -e fhir-parser ]; then
 	git submodule update --init
 fi
 
+pip3 install -r generator-requirements.txt
```

Comment:
I wouldn't add these commands as it may pollute somebody's environment if they haven't activated virtualenv and people may be using python2 (even though I'm not sure the generator still works with that).